.pre-commit-config.yaml

```diff
@@ -14,27 +14,27 @@ repos:
     - id: file-contents-sorter
       files: requirements-dev.txt
 
-- repo: https://gitlab.com/pycqa/flake8
-  rev: 3.7.9
-  hooks:
-    - id: flake8
-      exclude: docs/source/conf.py
-      args: [--max-line-length=105, --ignore=E203,E501,W503, --select=select=C,E,F,W,B,B950]
+#- repo: https://gitlab.com/pycqa/flake8
+#  rev: 3.9.1
+#  hooks:
+#    - id: flake8
+#      exclude: docs/source/conf.py
+#      args: [--max-line-length=105, --ignore=E203,E501,W503, --select=select=C,E,F,W,B,B950]
```

Comment:
Noting that flake8 has ben disabled. We will need to sort out the pre-commit at a later point so we can reduce contributor friction.

Reply:
Thanks to @NoraLoose for cleaning up my code and adding the warnings! I got pre-commit to pass, but not sure how. `isort` seemed to be unable to make up its mind about where things should be.